odbc/src/api/functions.rs

```diff
@@ -1259,17 +1408,20 @@ pub extern "C" fn SQLSetPos(
     _operation: USmallInt,
     _lock_type: USmallInt,
 ) -> SqlReturn {
+    dbg!();
     unsupported_function(MongoHandleRef::from(statement_handle), "SQLSetPos")
 }
 
 #[no_mangle]
 pub extern "C" fn SQLSetEnvAttr(
```

Comment:
I thought we decided to just provide a Unicode driver and only implemented the Unicode (W) functions. Did you add it because it is very simple? And should we add it, because it could make things a bit confusing if we sometimes support ANSI functions, not other times.